build.gradle

```diff
@@ -18,11 +10,11 @@ plugins {
 	id 'org.owasp.dependencycheck' version '5.2.4'
 	id 'com.github.ben-manes.versions' version '0.27.0'
 	id 'org.sonarqube' version '2.8'
-	id 'info.solidsoft.pitest' version '1.4.6'
+	id 'info.solidsoft.pitest' version '1.5.1'
 }
 
 group = 'uk.gov.hmcts.reform.mi'
-version = '3.2.1'
+version = '3.3.0'
```

Comment:
3.2.2 maybe?

Reply:
Could be, I was thinking the NotificationOutput change was  big enough of a breaking change it should be a minor upgrade rather than patch. Thoughts?